lib/govuk_publishing_components/presenters/meta_tags.rb

```diff
@@ -98,6 +99,12 @@ def add_step_by_step_tags(meta_tags)
         stepnavs = links[:part_of_step_navs] || []
         stepnavs_content = stepnavs.map { |stepnav| stepnav[:content_id] }.join(",")
         meta_tags["govuk:stepnavs"] = stepnavs_content if stepnavs_content.present?
+
+        step_nav_helper = PageWithStepByStepNavigation.new(content_item, request.path, request.query_parameters)
+        if step_nav_helper.show_secondary_step_by_step?
+          meta_tags["govuk:secondary-stepnav-shown"] = step_nav_helper.secondary_step_by_step[:content_id]
+        end
```

Comment:
I'd consider doing all the setup at the beginning of the method and doing the meta_tags assignment at the end of the function, using one-line-ifs for both for consistency:

```ruby
def add_step_by_step_tags(meta_tags)
  stepnavs = links[:part_of_step_navs] || []
  stepnavs_content = stepnavs.map { |stepnav| stepnav[:content_id] }.join(",")
  stepnav = PageWithStepByStepNavigation.new(content_item, request.path, request.query_parameters)

  meta_tags["govuk:stepnavs"] = stepnavs_content if stepnavs_content.present?
  meta_tags["govuk:secondary-stepnav-shown"] = stepnav.secondary_step_by_step[:content_id] if stepnav.show_secondary_step_by_step?
end
```

Reply:
The reason why I chose not to use inline ifs was that I wanted to improve the readability of the code. Where lines are more than 100 characters long, is it very difficult to read and follow (https://batsov.com/articles/2013/06/26/the-elements-of-style-in-ruby-number-1-maximum-line-length/ - this article goes into some detail about this if you're interested).

I've improved the logic a bit more